app/views/editor/verifier/VerifierTest.coffee

```diff
@@ -40,11 +40,22 @@ module.exports = class VerifierTest extends CocoClass
     me.team = @team = 'humans'
     @setupGod()
     @initGoalManager()
-    @register()
+    @fetchToken(@solution.source, @language)
+      .then((token) => @register(token))
+
+  fetchToken: (source, language) =>
+    if language not in ['java', 'cpp']
+      return Promise.resolve(source)
+
+    headers =  { 'Accept': 'application/json', 'Content-Type': 'application/json' }
+    m = document.cookie.match(/JWT=([a-zA-Z0-9.]+)/)
```

Comment:
Looks like this isn't used, does it need to be passed into the request or can we drop it?

Reply:
Left this due to how this may be used for developing with contractor.